react/SearchBreadcrumb.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react'
+
+import { BreadcrumbList } from './modules/breadcrumb'
+import { getBaseUrl } from './modules/baseUrl'
+
+interface SearchBreadcrumbItem {
+  name: string
+  href: string
+}
+
+const searchBreadcrumb = (
+  breadcrumb?: SearchBreadcrumbItem[]
+): BreadcrumbList | {} => {
+  if (!breadcrumb || !Array.isArray(breadcrumb)) {
```

Comment:
I think the `isArray` check is enough here 

```suggestion
  if (!Array.isArray(breadcrumb)) {
```